pages/content/amp-dev/documentation/guides-and-tutorials/start/create/include_image.md

```diff
@@ -2,19 +2,42 @@
 $title: Include an image
 $order: 2
 description: 'Most HTML tags can be used directly in AMP HTML, but certain tags, such as the <img> tag, are replaced with equivalent or slightly enhanced custom AMP HTML tags'
-author: pbakaus
-contributors:
-  - bpaduch
+author: crystalonscript
 ---
 
-Most HTML tags can be used directly in AMP HTML, but certain tags, such as the `<img>` tag, are replaced with equivalent or slightly enhanced custom AMP HTML tags (and a few problematic tags are outright banned, see [HTML Tags in the specification](../../../../documentation/guides-and-tutorials/learn/spec/amphtml.md#html-tags)).
+Most HTML tags can be used directly in AMP, but certain tags, such as the `<img>` tag, must be replaced with their AMP equivalents. See [HTML tags in the specification](https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/?format=websites#html-tags) for a full list.
 
-To demonstrate what additional markup could look like, here’s the code required to embed an image into the page:
+[tip type="note"]
+The AMP team is working to support native `<img>` tags on valid AMP pages. See [this GitHub issue](https://github.com/ampproject/amphtml/issues/30442) for more information and current progress.
+[/tip]
 
-[sourcecode:html]
-<amp-img src="welcome.jpg" alt="Welcome" height="400" width="800"></amp-img>
-[/sourcecode]
+We’ll add an image to our page, the AMP way, with the [`<amp-img>`](https://amp.dev/documentation/components/amp-img/?format=websites) tag. Copy and paste the code below into the `<body>`.
 
-[tip type="read-on"]
-**READ ON –** To learn why we’re replacing tags like `<img>` with [`<amp-img>`](../../../../documentation/components/reference/amp-img.md), and how many are available, visit [Include Images & Video](../../../../documentation/guides-and-tutorials/develop/media_iframes_3p/index.md).
-[/tip]
+`<amp-img src="https://amp.dev/static/samples/img/amp.jpg" width="680" height="410"></amp-img>` 
+
+AMP replaces certain default HTML tags for the following reasons:
```

Comment:
Idea: turn this paragraph into a notes section and combine with the paragraph about support for `img` tag.